src/varpubs/summarize_variants.py

```diff
@@ -44,11 +41,19 @@ def summarize_variants(
                 if not article:
                     continue
 
-                summary = summarizer.summarize(article)  # Summarize article content
-                output = f" Variant: {term}\n PMID: {pmid}\n Summary:\n{summary}\n🔗 DOI: {article.doi or 'N/A'}\n"
-                print(output)
-                all_summaries.append(output)
+                summary_text = summarizer.summarize(article)
+
+                row = [term, pmid, summary_text, article.doi or "N/A"]
+                rows.append(row)
+
+                logging.info(f"Variant: {term}")
+                logging.info(f"PMID: {pmid}")
+                logging.info(f"Summary:\n{summary_text}")
+                logging.info(f"DOI: {article.doi or 'N/A'}")
+                logging.info("-" * 40)
```

Comment:
Not really sure all this logging is really that useful in this place. I think we can just remove it here.

```suggestion
```